onnxruntime/core/providers/webgpu/shader_variable.cc

```diff
@@ -55,7 +60,11 @@ constexpr static const std::string_view VALUE_TYPE_ARRAY[] = {
     "u32",         // Uint8x4 (u32 as 4 elements of uint8)
     "vec2<u32>",   // Uint8x8 (vec2<u32> as 2x4 elements of uint8)
     "vec4<u32>",   // Uint8x16 (vec4<u32> as 4x4 elements of uint8)
-    "i32",         // Int8x4
+    "u32",         // Int8x4 (u32 as 4 elements of uint8)
+    "vec2<i32>",   // Int8x8 (vec2<i32> as 2x4 elements of int8)
+    "vec4<i32>",   // Int8x16 (vec4<i32> as 4x4 elements of int8)
+    "u32",         // Uint4x8
+    "i32",         // Int4x8
```

Comment:
should this be u32 for consistency?

also , there are other inconsistency. for example, Int8x4 uses 'u32' but Int8x8 uses 'vec2<i32>'.

Since there is no (u)int8/(u)int16 in WGSL, we always uses 4-bytes-integer as bit representation, maybe always using u32 is a good idea (RSHIFT is safer since no sign bit)

Reply:
done